web/src/pages/requests/index.tsx

```diff
@@ -348,6 +350,8 @@ const UserAccessCard: React.FC<{
             flexDir="column"
             key={req.id}
             pos="relative"
+            // id={"req_"+ index}
+            id={"req_" + req.reason}
```

Comment:
Can we use `data-testid` as an attribute rather than `id` here? To prevent potential ID collisions in HTML